test/client/components/ShareTopicModal.test.js

```diff
@@ -2,8 +2,8 @@ require('../support')
 import React from 'react'
 import { mount } from 'enzyme'
 import { configureStore } from '../../../src/store'
-import App from '../../../src/containers/App.js'
```

Comment:
I am assuming this ".js" thing was a legacy thing and no longer necessary even in the server context? It of course breaks the App/index.js default thing relied upon for the pod.